Makefile

```diff
@@ -80,6 +80,10 @@ fmtcheck:
 fmt:
 	gofmt -w $(GOFMT_FILES)
 
+spellcheck:
+	@echo "==> Spell checking website..."
+	@misspell -error -source=text website/source
```

Comment:
I think we should add misspell to the external tools so we can ensure it's installed

Reply:
Fixed.